packages/nest/src/schematics/graphql/schema.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
How about if the `@nrwl/nest:application` schematic supported a `protocol` flag that supported the following options right now?
* Http
* GraphQL

This could be expanded in the future to include things like WebSocket and gRPC.

Users would be able to create a graphql application via:

```
ng g @nrwl/nest:app graphql-app --protocol graphql
```

Users could also set this as a default if all of their backends should use a certain protocol by default.

:thinking:

Reply:
Most GraphQL API's still have some REST functionality enabled, for doing things like oAuth redirects, webhooks or uptime checks. Also, it relies on having Nest with either `express` or `fastify`.

With that in mind I think it makes most sense to just add a `--graphql`  boolean to the `@nrwl/nest:app` schematic.